user_manual/pim/sync_osx.rst

```diff
@@ -2,16 +2,51 @@
 Synchronizing with macOS
 ========================
 
-With Nextcloud 14+ and its CalDAV-backend, syncing your Calendars with your macOS-device is easy.
+Setup your Accounts
+-------------------
 
-To use Nextcloud with macOS Calendar you will need to use the following steps:
+Adding your Calendar and Contacts resources from Nextcloud Groupware to macOS
+native applications is easy. In the following steps you will add your server
+resources for **CalDAV** (Calendar) and **CardDAV** (Contacts) to your Nextcloud.
```

Comment:
It may not be welcoming to new users to say something is easy – I'm not a new user and the reason I'm reviewing this is because it wasn't easy for me :)
I don't know for sure what would be better, unfortunately. I do think that with these explicit steps it will be easier.